<launch>
  <node name="camera_values_node" pkg="ucsd_robo_car_simple_ros" type="camera_values_ros.py" />
  <node name="ros_racer_calibration_node" pkg="ucsd_robo_car_simple_ros" type="ros_racer_calibration_node.py" />
</launch>
